README.md

```diff
@@ -1,2 +1,268 @@
-# Hello-Microverse
-This project serves to show the understanding of version control and linters
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>README</b></h3>
+</div>
+
+<br>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+
+Hello Microverse is the first miniproject from module 1. This project serves to familiarize students with Git, Github,gitflow and linters.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+  
+  <ul>
+    <li><a href="https://html.spec.whatwg.org/">HTML</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">CSS</a></li>
+  </ul>
+
+
+### Key Features <a name="key-features"></a>
+
+- **Displays the text "Hello Microverse in my favourite color**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+A browser of your choosing.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+
+### Install
+
+Install this project with:
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+
+### Deployment
+
+You can deploy this project using:
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 Andrew Mccollin
+
+- GitHub: [@githubhandle](https://github.com/Drew246)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/andrew-mccollin)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+ - N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
```

Comment:
- [ ] You have done great customizing the readme file, but there are still some changes you need to make here before moving to the next milestone. :smile: 
- [ ] As required by Microverse, you need to fill in each section of the readme. looking at the screenshot below, you can see that these sections haven't been filled up. I'll suggest you add the correct content there. :smile: :+1: . Also ill suggest you fill in properly how to get your project to run locally. :wave: 

- - Screenshot:
  
![Selection_077](https://github.com/Drew246/Hello-Microverse/assets/107632780/9bc10a6e-f867-470f-9a3d-ce3b1cc6d0e1)

- [ ] Also it seems your readme file is duplicated, I'll suggest you correct this too.